.github/workflows/deploy.yaml

```diff
@@ -1,24 +1,25 @@
-name: Deploy to production
+name: Deploy to Production
 
 on:
   push:
     branches:
       - master
 
 jobs:
-  build:
+  deploy:
+    name: Deploy
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - name: Login to Heroku Container registry
-      env:
-        HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
-      run: heroku container:login
-    - name: Build and push
-      env:
-        HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
-      run: heroku container:push -a ${{ secrets.HEROKU_APP_NAME }} web
-    - name: Release
-      env:
-        HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
-      run: heroku container:release -a ${{ secrets.HEROKU_APP_NAME }} web
\ No newline at end of file
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Deploy to Heroku
+        uses: akhileshns/heroku-deploy@v3.12.12
+        with:
+          heroku_api_key: ${{ secrets.HEROKU_API_KEY }}
+          heroku_app_name: "boonafide"
+          heroku_email: ""
+          usedocker: true
+        env:
+          HD_DBURL: ${{ secrets.DBURL }}
+          HD_NODE_ENV: production
+          HD_PORT: 3001
```

Comment:
Do we actually need to set the port?

Reply:
I am going to merge (I don't know how to test it locally) the branch `ana/ci`, including the Deploy part, but I am trying to take the env from Heroku with:

```
env:
        HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
```

Reply:
It worked!!!!!! TY so much!
Deploy [file](https://github.com/aemartos/boonafide/pull/6/files#diff-4f9e38227ed64fefb17f4668a7ac4ab55b6149994d5ac2fd96182d5958479b54).